packages/api/src/modules/auth/types.ts

```diff
@@ -72,6 +72,9 @@ export interface ISignInResponse {
   expired_at: Date;
   avatar: string;
   user_id: string;
```

Comment:
don`t need this infos

after create a new column to link a vault 1:1 on user column, return id of this vault on request, to redirect user on UI